src/backend/efiling-api/src/main/resources/application.yml

```diff
@@ -23,6 +23,7 @@ jag:
       demo: ${DEMO_MODE:false}
     navigation:
       baseUrl: ${NAVIGATION_BASE_URL:http://localhost:3000/efiling}
+      csoBaseUrl: ${CSO_BASE_URL:http://localhost/cso}
```

Comment:
shouldnt this be:
https://dev.justice.gov.bc.ca/cso

Reply:
not the greatest, will have to think about it lol